README.rst

```diff
@@ -79,6 +79,13 @@ Every time you develop something in this repo
 
   # Open a PR and ask for review.
 
+Setting up an exam and proctoring tool
```

Comment:
This looks like what Zach already committed. Are you working off the latest version of `main`? Don't forget to rebase!

Reply:
If this looks incorrect then I probably bungled rebasing off main somehow

Reply:
Let me know if you need any help figuring this one out. You could fix the typo in `converterd` while you're in here, though! 😄

Reply:
Git stuff has been resolved, this is no longer present in the PR